addOns/quickstart/src/main/java/org/zaproxy/zap/extension/quickstart/AttackPanel.java

```diff
@@ -392,6 +394,8 @@ private void stopAttack() {
         getExtensionQuickStart().stopAttack();
 
         stopButton.setEnabled(false);
+        getSpiderCheckBox().setEnabled(true);
+        plugableSpider.setEnabledAjax(true);
```

Comment:
Should be done in `notifyProgress` for all possible cases (might worth extracting a method, to change the state of the buttons and the checkboxes).

Reply:
I think that’s okay, they’re subject to the same conditions.